After exporting PYTHONPATH which indicates to /usr/lib/python*/site-packages, if we use a semicolon
on the same line with an import statement, interpreter raises error.
See followings to explore more:

https://bugs.archlinux.org/task/22823
https://issues.rpath.com/browse/CNY-3178

Index: istanbul-0.2.2/configure.ac
===================================================================
--- istanbul-0.2.2.orig/configure.ac
+++ istanbul-0.2.2/configure.ac
@@ -74,7 +74,7 @@ PKG_CHECK_MODULES(PYGST, gst-python-0.10
       [AC_MSG_RESULT([$PYGST_PKG_ERRORS])])
 
 saved_PYTHONPATH=$PYTHONPATH
-export PYTHONPATH=$PYGST_DIR:$PYTHONPATH
+export PYTHONPATH=
 AS_PYTHON_IMPORT([gst],,
     [AC_MSG_ERROR([Unable to import gst-python 0.10 -- check your PYTHONPATH?])
     GST_SUPPORTED=no],
